packages/jasmine-parameterized/src/parameterizeSpec.ts

```diff
@@ -0,0 +1,51 @@
+import { parameterize } from './parameterize.js';
```

Comment:
for all these files, parameterizeSpec.ts, parameterizeSuite.ts, parameterizeSpec.spec.ts, parameterizeSuite.spec.ts
can you switch to the kebab case convention? parameterize-spec.ts, parameterize-suite.ts, parameterize-spec.spec.ts, parameterize-suite.spec.ts